css-two-column-layout/index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Two Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="column-full">
+        <header>
+          <h1>SCALE, SPEED, AGILITY</h1>
+          <p class="center-align">We've dedicated thousands of man hours to making healthcare institutions in the Helium <br> network run 10x more efficiently and deliver an unparalleled quality of care.</p>
+        </header>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-one-half">
+        <img src="images/software.svg" alt="software image">
+        <h2>INTUITIVE, SMART SOFTWARE</h2>
+      </div>
+      <div class="column-one-half">
+        <img src="images/practice.svg" alt="practice image">
+        <h2>PRACTICE MEDICINE, NOT ADMINISTRATION</h2>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-one-half">
+        <p class="textpos">
+          We build products that transform how healthcare facilities run across Africa - making them radically more efficient operationally and financially, a complete turnaround.</p>
+      </div>
+      <div class="column-one-half">
+        <p class="textpos">
+          Our all-in-one electronic health records (EHR), hospital management (HMIS) &amp; billing system helps streamline your facility, reduce waste and improve quality of care.
+        </p>
+      </div>
+    </div>
```

Comment:
Try restructuring your HTML so that a column holds the picture, the heading, and the text all in one.